### PR TITLE
fix: edit the csv file should trigger an update of knowledgebase

### PR DESCRIPTION
**Pull request by 0xff-dev**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it
fix: edit the csv file should trigger an update of knowledgebase

#### Which issue(s) this PR fixes

Fixes #

### CODE REVIEW

**Comment by Abirdcfly** [comment]
**Just a suggestion**: 

we can change the direct modification of the csv in the test to modifying the csv through the api to make sure the whole process is working correctly:

https://github.com/kubeagi/arcadia/blob/00dd4b9071e6b7733f5ba5490502ef4aa85b164d/tests/example-test.sh#L482-L487

File: apiserver/pkg/common/read_csv.go [hunk 1]
```diff
@@ -43,9 +43,10 @@ type (
 		// Do you want to force an update?
 		ForceUpdate bool `json:"forceUpdate"`
 
-		UpdateLines []CSVLine  `json:"updateLines"`
-		NewLines    [][]string `json:"newLines,omitempty"`
-		DelLines    []int      `json:"delLines"`
+		UpdateLines   []CSVLine  `json:"updateLines"`
+		NewLines      [][]string `json:"newLines,omitempty"`
+		DelLines      []int      `json:"delLines"`
+		Knowledgebase string     `json:"knowledgebase,omitempty"`
```
Comment (bjwswang) [inline apiserver/pkg/common/read_csv.go:49]: @0xff-dev There may have more than one knowledgebases which has this csv

**Comment by bjwswang** [comment]
@0xff-dev as discussed, please add verion in filegroup and update the version once edit succ

**Review by bjwswang — APPROVED** [review]

**Comment by bjwswang** [comment]
@0xff-dev 
Please remove the explicitly content check in modelscope_test.go

**Pull request closed without merging**